voice/gradle.properties

```diff
@@ -0,0 +1,2 @@
+POM_NAME=InstantSearch Android Voice
```

Comment:
I'm not sure what is this variable for, but shouldn't it be `InstantSearch Android VoiceOverlay`?

Reply:
```suggestion
POM_NAME=InstantSearch Android Voice Overlay
```
it's for the metadata, nice catch :)